solr_conf/conf/solrconfig.xml

```diff
@@ -35,16 +35,86 @@
       <str name="fl"> *, score</str>
 
       <str name="qf">
-        id
-        descriptive_tiv
+        sw_display_title_tesim^5
```

Comment:
make the default qf here match what is in blacklight_config_helper qf (the latter of these is what is actually used for qf)

Reply:
See PR sul-dlss/sul-solr-configs/pull/281 for more info on these changes